ocp_resources/resource.py

```diff
@@ -446,16 +450,53 @@ def to_dict(self):
 
     def __enter__(self):
         signal(SIGINT, self._sigint_handler)
-        return self.deploy()
+        """
+        For debug, export REUSE_IF_RESOURCE_EXISTS to skip resource create.
```

Comment:
could this be an `__init__` variable instead of environment?

Reply:
No, the point of this is to let the user control it without any code change.